apps/web/src/store/modals.ts

```diff
@@ -1,11 +1,22 @@
 import type { Profile, Publication } from '@lenster/lens';
 import { create } from 'zustand';
 
+export enum PublicationTypes {
```

Comment:
rename to something different, cuz `PublicationTypes` is already an enum in lens package